### ISSUE TITLE
Release to App Store(s) with Travis

### ISSUE THREAD

**Issue by lathonez** [lathonez/clicker#43]
Not sure if this is even possible atm, but without it CI / CD is a little moot.

1. Release to play store to understand the process
2. Try to incorporate that in to Travis
3. Repeat for app store
4. Try to incorporate that in to Travis
5. Set up build matrix in Travis to do both app store and play store

**Comment by BugsBunnyBR** [comment]
Hey, since it is ionic, maybe support the deploy using ionic.io, the deploy and packaging seems to be nice features. If their build system support tests I don't know... maybe there is the need to run travis and afterwards send the code to ionic.io...

**Comment by lathonez** [comment]
Yes, thank you. I'd like everything to be integrated nicely into a CI/CD workflow and Travis seems as good as any for that.
